usaspending_api/accounts/views/financial_spending.py

```diff
@@ -24,7 +23,7 @@ def get_queryset(self):
 
         # required query parameters were not provided
         if not (fiscal_year and agency_id):
-            raise InvalidParameterException('Missing one or more required query parameters: fiscal_year, agency_id')
+            raise ParseError('Missing one or more required query parameters: fiscal_year, agency_id')
```

Comment:
Heh, saw you removed `ParseError` in a previous commit, and now it's back. Is there a reason not to use `InvalidParameterException` here? I do think we should be returning a 400 if the request is invalid.